composer.json

```diff
@@ -7,15 +7,14 @@
         "monolog/monolog": "^1.17",
         "neos/utility-files": "^3.0",
         "padraic/phar-updater": "1.0.5",
-        "symfony/console": "^3.4.17 || ^4.0",
-        "symfony/process": "^3.0 || ^4.0",
-        "symfony/options-resolver": "^3.0 || ^4.0",
+        "symfony/console": "^4.0",
```

Comment:
I would even go higher. In the meantime ^5.0 is the current version.

here is my list:

```
"require": {
        "php": ">=7.1 <7.4",
        "guzzlehttp/guzzle": "^6.5",
        "monolog/monolog": "^2.0",
        "neos/utility-files": "^6.1",
        "padraic/phar-updater": "^1.0",
        "paragonie/random_compat": "^2.0 || ^9.99",
        "symfony/console": "^5.0",
        "symfony/options-resolver": "^5.0",
        "symfony/process": "^5.0",
        "webmozart/assert": "^1.7"
    },
    "require-dev": {
        "friendsofphp/php-cs-fixer": "^2.14",
        "phpunit/phpunit": "^8.5",
        "sphpdox/sphpdox": "dev-master",
        "symfony/finder": "^5.0"
    },
```

Reply:
@simonschaufi Yes, but this versions are tested in the wild. So i would remain conservative here. For example with phpunit 8.5 we would have to refactor the test classes. In the near future i will do it, but for now i would prefer the current versions.

Reply:
ok, then leave phpunit but how about the rest?

Reply:
That´s true. But still we have to test it. Can we do it next week? For now i would just raise the current highest versions. I bet we will go higher as sooner than later.

Reply:
I‘m fine with symfony ^4.4 for now

Reply:
ok. 2:1 :smile: